.github/workflows/deploy.yml

```diff
@@ -119,12 +119,12 @@ jobs:
 
       - name: Update Cursor File
         id: cursor-update
-        uses: pleo-io/pleo-spa-cicd/actions/cursor-deploy@v6.0.0
+        uses: pleo-io/pleo-spa-cicd/actions/cursor-deploy@spa-github-actions-v8.0.2
         with:
           bucket_name: ${{ inputs.bucket_name }}
 
       - name: Update PR Description
-        uses: pleo-io/pleo-spa-cicd/actions/post-preview-urls@v6.0.0
+        uses: pleo-io/pleo-spa-cicd/actions/post-preview-urls@spa-github-actions-v8.0.2
```

Comment:
@pekala I didn't realize we have some dependencies to our own package. We should probably ensure that these use the latest version.